packages/okta-react/test/e2e/harness/src/App.js

```diff
@@ -29,8 +29,9 @@ class App extends Component {
         <Router>
           <Security issuer={process.env.REACT_APP_ISSUER}
                     client_id={process.env.REACT_APP_CLIENT_ID}
+                    disableHttpsCheck={true}
```

Comment:
Probably ok for testing, but did we not get `process.env. DISABLE_HTTPS_CHECK` to register here?

Reply:
We did. Let me update it.

Reply:
For some weird reason, it's not picking it up. I'm going to leave it like this for now.